pkg/azurefile/azurefile.go

```diff
@@ -313,6 +319,51 @@ func (d *Driver) expandVolume(ctx context.Context, volumeID string, capacityByte
 	return volumehelper.GiBToBytes(int64(resp.Quota())), nil
 }
 
+func (d *Driver) createVHD(ctx context.Context, volumeID, accountName, accountKey, fileShareName, diskName string) error {
+	shareURL, err := d.getShareURL(volumeID)
+	if err != nil {
+		return fmt.Errorf("failed to get share url with (%s): %v", volumeID, err)
+	}
+	properties, err := shareURL.GetProperties(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to get properties of share(%v): %v", shareURL, err)
+	}
+	sizeBytes := volumehelper.GiBToBytes(int64(properties.Quota() - 1))
```

Comment:
make this `sizeBytes` as a parameter, `properties.Quota() - 1` is hard to read

Reply:
done